spec/orders.mediawiki

```diff
@@ -831,6 +831,64 @@ client.
 | orderid || string || order ID
 |}
 
+A client can request the current match status using the
+<code>match_status</code> request. Note that the route is provided as a
+convenience, and persistence of match data may be subject to the operator's
+archiving policy. <code>match_status</code> can be used to recover after a
+temporary disconnection, but should not be relied on as a source of historical
+match data.
+
+'''Notification route:''' <code>match_status</code>, '''originator:''' client
+
+<code>payload</code>
+{|
+! field    !! type     !! description
+|-
+| matches  || [<code>MatchRequest</code>] || an array of <code>MatchRequest</code> objects
+|}
+
+'''<code>MatchRequest</code> object'''
+
+{|
+! field   !! type   !! description
+|-
+| base    || int    || the base asset ID
+|-
+| quote   || int    || the quote asset ID
```

Comment:
Were you thinking to include base/quote to reduce possible query burden on the server?

Reply:
Yes. Since we don't have a single db index for the match tables, a manual search would scale poorly with more markets, I think.